tests/csm/rest/test_capacity_quota.py

```diff
@@ -298,18 +324,24 @@ def test_40635(self):
         enabled = test_cfg["enabled"]
         max_size = test_cfg["max_size"]
         max_objects = test_cfg["max_objects"]
-        payload = self.csm_obj.iam_user_quota_payload(enabled,max_size,max_objects)
+        payload = self.csm_obj.iam_user_quota_payload(enabled,max_size,max_objects,
+                              check_on_raw=True)
         self.log.info("Step 2: Perform PUT API to set user level quota with max values")
         result, resp = self.csm_obj.verify_get_set_user_quota(self.user_id, payload,
                                                                verify_response=True)
         assert result, "Verification for get set user failed."
         self.log.info("Response : %s", resp)
         self.log.info("Step 3: Perform max size verification")
-        res = self.csm_obj.verify_max_size(max_size, self.akey, self.skey, self.bucket)
-        assert_utils.assert_false(res[0], res[1])
-        self.log.info("Step 4: Delete object")
-        assert s3_misc.delete_object(
-            self.obj_name, self.bucket, self.akey, self.skey), "Failed to delete bucket."
+        res, obj_list = self.csm_obj.verify_max_size(max_size, self.akey, self.skey,
+                                     self.bucket)
+        assert True, res[1]
```

Comment:
What is the use of this assert True? Can you please correct this?

Reply:
corrected